presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -532,6 +652,217 @@ public static double round(@SqlType(StandardTypes.DOUBLE) double num, @SqlType(S
         return Math.round(num * factor) / factor;
     }
 
+    private static SqlScalarFunction decimalRoundFunction()
```

Comment:
round, truncate and sign got mixed up here during some fixup squashing ... I will untangle this, but mod is already in it's own commit, so that should be OK